proposals/0133-syscall-get-epoch-stake.md

```diff
@@ -67,32 +76,44 @@ uint64_t sol_get_epoch_stake(/* r1 */ void const * vote_addr);
 
 ### Control Flow
 
-The syscall aborts the virtual machine if not all bytes in VM memory range
-`[vote_addr, vote_addr + 32)` are readable.
+If `var_addr` is _not_ a null pointer:
 
-Otherwise, the syscall returns a `u64` integer representing the total active
-stake delegated to the vote account at the provided address.
+- The syscall aborts the virtual machine if not all bytes in VM memory range
+  `[vote_addr, vote_addr + 32)` are readable.
+- Otherwise, the syscall returns a `u64` integer representing the total active
+  stake delegated to the vote account at the provided address.
+  If the provided vote address corresponds to an account that is not a vote
+  account or does not exist, the syscall will return `0` for active stake.
 
-If the provided vote address corresponds to an account that is not a vote
-account or does not exist, the syscall will return `0` for active stake.
+If `var_addr` is a null pointer, the syscall returns a `u64` integer
+representing the total active stake on the cluster for the current epoch.
 
 ### Compute Unit Usage
 
-The syscall will always attempt to consume the same amount of CUs regardless of
-control flow.
+The syscall will always attempt to consume the same amount of CUs for both
+possible `var_addr` pointers, regardless of control flow.
```

Comment:
This sentence is confusing, since the following lines define *different* amounts of CUs for the two possible `var_addr` pointers. Do we need this sentence? If so, can you reword it more simply?

Reply:
Updated it to be more clear!

Reply:
Much better, thanks!